json_reader.py

```diff
@@ -0,0 +1,92 @@
+import json
+from Person import Person
+
+class JSONReader(object):
+    path = 'jsonFile.json'
+
+    def get_all(self):
+        person_details = list()
+        with open(self.path) as data_file1:
+            data = json.load(data_file1)
+            if data == {}:
+                print 'json data is empty'
```

Comment:
There shouldn't be any print statements in the Persistence layer. Later if you reuse the same project to work as a Web app, think of what happens to the print statements.